Initial

glibin opened the pull request.

Commit:
init commit

Commit:
Add huntflow data converter

Commit:
Fix: converting, connecting, docker Add: util

Commit:
All is done

Commit:
End of develop

Commit:
minor changes

Commit:
minor changes

Commit:
minor changes

Commit:
Инструкция по запуску.odt

Commit:
FIX Инструкция по запуску.odt

Commit:
Убрал токен

glibin commented on .gitignore:
```diff
@@ -1,10 +1,17 @@
+<<<<<<< HEAD
```
нужно поправить конфлик

DriveAthlete replied:
Добрый день, поправил .gitignore. Не заметил в момент решения конфликта слияния.

glibin commented on config/settings/base.py:
```diff
@@ -0,0 +1,271 @@
+# Имя типа веб хука, по которому разрешается доступ (берется из переменных среды)
+ALLOWED_TYPE = env.str("ALLOWED_TYPE")
+# Шаблон ссылки на вакансию
+VACANCY_BASE_URL = "https://huntflow.ru/my/{nick}#vacancy/"
```
должна быть ссылка на кандидата (обсуждали на демо) на вакансии

DriveAthlete replied:
Добрый день, это базовая ссылка на вакансию, которая используется для генерации ссылки на кандидата в вакансии. В `src/connectors/converters.py` в методе `_get_vacancy_url` генерируется требуемая ссылка по следующему шаблону `"{url}{vacancy_id}/filter/workon/id/{applicant_id}"`.

Commit:
Fix .gitignore

Commit:
Убранно sentry

Commit:
Исправлен баг: отсутствие категории вакансии в ответе веб-хука
если у кандидата не приходит в веб-хуке категория вакансии, то возвращаем None в поле с катекгорией вакансии

Commit:
Изменение инструкции по запуску
